### PR TITLE
Email fix and endpoint

### PR DESCRIPTION
**Pull request by Pelozo**
And I say, hey hey hey hey
I said hey, what's going on?

### CODE REVIEW

File: tasks/views.py [hunk 1]
```diff
@@ -30,19 +33,36 @@ def get_context_data(self, **kwargs):
...
-        contexto = {
-            'form': form
-        }
-    else:
-        form =TaskForm(request.POST)
-        contexto = {
-            'form': form
-        }
-        if form.is_valid():
-            form.save()
```
Comment (Silvania19) [inline tasks/views.py:46]: era codigo de recuerdo compa

File: user/views.py [hunk 2]
```diff
@@ -33,8 +35,8 @@ class Register(CreateView):
 
     def get_success_url(self):
         # send welcome email
-        msg_plain = render_to_string('emails/welcome.txt', {'params': self.object})
-        msg_html = render_to_string('emails/welcome.html', {'params': self.object})
+        msg_plain = render_to_string('emails/welcome.txt', {'name': self.object.first_name})
+        msg_html = render_to_string('emails/welcome.html', {'name': self.object.first_name})
         send_mail(
```
Comment (Silvania19) [inline user/views.py:38]: Buen trabajo

**Review by Silvania19 — APPROVED** [review]

File: tasks/views.py (hunk 1 above)
Comment (Pelozo) [reply tasks/views.py:46]: :sunglasses:

File: user/views.py (hunk 2 above)
Comment (Pelozo) [reply user/views.py:38]: :dolphin: :hammer: :dolphin: :hammer: :dolphin: :hammer: :toilet: :hammer: :toilet: :hammer: :toilet:

**Pull request merged**